File: two-pointer/26-Remove-Duplicates-from-Sorted-Array.go
func removeDuplicates(nums []int) int {
    found := 0
    for i := 1; i < len(nums); i++ {
        if nums[found] != nums[i] {
            found++
            nums[found], nums[i] = nums[i], nums[found]
        }
    }
    return found + 1
}

File: two-pointer/26-Remove-Duplicates-from-Sorted-Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        found = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[found]:
                found += 1
                nums[found], nums[i] = nums[i], nums[found]
        return found + 1